tests/solvers/test_radau5.py

```diff
@@ -1732,47 +1850,33 @@ def test_step_size_too_small(self):
         prob = Implicit_Problem(f, y0, yd0)
 
         sim = Radau5DAE(prob)
-        sim.solver = 'c'
 
         sim.atol = 1.e10
         sim.rtol = 1.e10
 
         sim.inith = 1.e-1
         sim.maxh = 1.e-1
         
-        nose.tools.assert_raises(Radau5Error, sim.simulate, 1. + 1.e-16)
+        err_msg = f"The step size became too small. At time {float_regex}."
+        with nose.tools.assert_raises_regex(Radau5Error, err_msg):
+            sim.simulate(1. + 1.e-16)
 
     @testattr(stddist = True)
     def test_repeated_unexpected_step_rejections(self):
         """
-        This tests the error for repeated unexpected step rejections
+        This tests the error for repeated unexpected step rejections in Radau5DAE.
         """
-        pass
-        # def f(t, y, yd):
-        #     raise N.linalg.LinAlgError()
-        # prob = Implicit_Problem(f, N.array([1.]), N.array([1.]))
-        # sim = Radau5DAE(prob)
-        # sim.solver = 'c'
-
-        # nose.tools.assert_raises(Radau5Error, sim.simulate, 1.)
-
-    @testattr(stddist = True)
-    def test_keyboard_interrupt_fcn(self):
-        """Test that KeyboardInterrupts in right-hand side terminate the simulation. Radau5 + C + implicit problem."""
-
-        y0 = np.array([1., 1.])
-        yd = np.array([0., 0.])
-        aux = KeyboardInterruptAux(dim = len(y0), fcn = True)
-        prob = Implicit_Problem(aux.f_impl, y0, yd)
+        def f(t, y, yd):
+            raise N.linalg.LinAlgError()
+        prob = Implicit_Problem(f, N.array([1.]), N.array([1.]))
         sim = Radau5DAE(prob)
-        sim.solver = 'c'
 
-        err_msg = "Unrecoverable exception encountered during callback to problem (right-hand side/jacobian)."
-        with nose.tools.assert_raises_regex(Radau5Error, re.escape(err_msg)):
+        err_msg = f'Repeated unexpected step rejections.'
```

Comment:
You are using an f-string here but not formatting the string, I am just adding the comment here in case you missed something.